<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotationMeasure</class>
 <widget class="QDialog" name="RotationMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="filePath"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="fileButton">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="stairPath"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="stairButton">
        <property name="text">
         <string>Stair path</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSpinBox" name="top">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Top point</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSpinBox" name="bottom">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bottom point</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QSpinBox" name="module"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Module</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QSpinBox" name="ray"/>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ray</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RotationMeasure</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RotationMeasure</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
